packages/grid-pro/README.md

```diff
@@ -1,13 +1,12 @@
-# &lt;vaadin-grid-pro&gt;
+# @vaadin/grid-pro
 
-[Live Demo ↗](https://vaadin.com/components/vaadin-grid-pro/html-examples)
-|
-[API documentation ↗](https://vaadin.com/components/vaadin-grid-pro/html-api)
+> ℹ️ A commercial Vaadin [subscription](https://vaadin.com/pricing) is required to use Grid Pro in your project.
```

Comment:
Description and subscription hint are out of order.

Reply:
Fixed.